src/Controller/EnMarche/NewsletterController.php

```diff
@@ -114,4 +115,17 @@ public function invitationSentAction(Request $request)
             'invitations_count' => $invitationsCount,
         ]);
     }
+
+    public function renderNewsletterFormAction(array $options): Response
```

Comment:
I think we are going in the wrong way with this. Let's create a twig extension instead. It would avoid the overhead of sub requests and we will not end with view logic inside a controller.

Any thoughts @illusionOfParadise @ottaviano @Remg @thomasbisignani ?

Reply:
I was my first Idea but with some discussion with salah and other people I took this way :s